.idea/codeStyles/Project.xml

```diff
@@ -3,9 +3,18 @@
     <JetCodeStyleSettings>
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value>
-          <package name="java.util" withSubpackages="false" static="false" />
-          <package name="kotlinx.android.synthetic" withSubpackages="true" static="false" />
-          <package name="io.ktor" withSubpackages="true" static="false" />
+          <package name="java.util" alias="false" withSubpackages="false" />
```

Comment:
I don't think this file is needed

Reply:
I reverted the change to this file.
Should we delete the .idea directory and add it to .ignore ?
If yes, it will be better to do it in a more global PR, for example for cleaning the whole project.